README.md

```diff
@@ -12,20 +12,20 @@ fits within the selected maximum line-length.
 
 For example, the following snippet:
 
-```erl
+```erl formatted hello
```

Comment:
I really like how this works! And I haven't thought about checking readme either. Good job!

Reply:
It turns out the use case I needed was already there :)
I also stumbled upon this and decided to use this and then we can add more markdown files in doc and root and they will automatically be checked from the get go.